tests/fsharp/Compiler/Language/OpenTypeDeclarationTests.fs

```diff
@@ -1055,17 +1078,53 @@ module Test =
 
         static member M() = ()
 
-open type Test.TestRecord<int>
+open Test
 
 module Test2 =
 
     let x = { X = "" }
 
+open type Test.TestRecord<int>
+
+module Test3 =
+
+    let x = { X = "" }
+
     let y = M()
         """
         |> withLangVersionPreview
         |> compile
-        |> shouldFail
+        |> withErrorCode 1
+        |> ignore
+
+    [<Test>]
+    let ``Open generic record should have access to pattern record via labels with enclosing type instantiations`` () =
```

Comment:
This and the others are good tests. The name of the test kind of implies that something will "work", but you're validating that it actually fails due to a type mismatch. Should that be in the name of these test cases?

Reply:
Would be a good idea.